.specify/extensions/git/scripts/bash/create-new-feature-branch.sh

```
@@ -591,6 +539,37 @@ if [ "$CREATE_WORKTREE" = true ]; then
+            jq -cn --arg n "$FEATURE_NUM" '{feature_number:$n}' > "$fj" 2>/dev/null && return 0
+        fi
+    fi
+    # No jq: best-effort fallback on compact single-line JSON only; leave the
+    # file untouched on any doubt.
+    if [ -f "$fj" ] && [ "$(wc -l < "$fj" | tr -d ' ')" -le 1 ]; then
+        if grep -q '"feature_number"' "$fj" 2>/dev/null; then
+            sed -E 's/"feature_number"[[:space:]]*:[[:space:]]*[^,}]*/"feature_number":"'"$FEATURE_NUM"'"/' "$fj" > "$fj.tmp" 2>/dev/null \
+                && mv -f "$fj.tmp" "$fj" || rm -f "$fj.tmp"
+        elif grep -q '}[[:space:]]*$' "$fj" 2>/dev/null; then
+            sed -E 's/}[[:space:]]*$/,"feature_number":"'"$FEATURE_NUM"'"}/' "$fj" > "$fj.tmp" 2>/dev/null \
+                && mv -f "$fj.tmp" "$fj" || rm -f "$fj.tmp"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)